src/Options.jsx

```diff
@@ -123,6 +123,8 @@ class Options extends React.Component {
           value={(pageSize || pageSizeOptions[0]).toString()}
           onChange={this.changeSize}
           getPopupContainer={(triggerNode) => triggerNode.parentNode}
+          aria-label={locale.page_size}
+          defaultOpen={false}
```

Comment:
```suggestion
```

`defaultOpen={false}` could be deleted.

Reply:
We need defaultOpen to set the `Input` components accessibility issue. For the detail you can check my this comment: https://github.com/ant-design/ant-design/issues/31692#issuecomment-914906885

Reply:
@afc163 Any other problems here?